src/controllers/csi/gc/unmounted.go

```diff
@@ -72,7 +72,9 @@ func determineMaxUnmountedVolumeAge(maxAgeEnvValue string) time.Duration {
 		return defaultMaxUnmountedCsiVolumeAge
 	}
 	if maxAge < 0 {
-		return 0
+		maxAge = 0
```

Comment:
we could keep the return here and add a different log here stating that files will be deleted "instantly" (on next garbage collection), since stating a max age of 0 days could be up to interpretation